app/models/user.rb

```diff
@@ -25,4 +25,8 @@ def self.find_or_create_from_auth_hash(auth_hash)
       user.provider = provider
     end
   end
-end
\ No newline at end of file
+
+  def admin?
```

Comment:
Don't need this anymore if we're not able to get advanced constraints to work through class methods on constraints. See https://guides.rubyonrails.org/routing.html#advanced-constraints

Reply:
Eventually when we need to build in admin features or access control for customer data, we'll setup `Pundit` and start writing policies